org.eclipse.lyo.tools.adaptormodel.edit/plugin.properties

```diff
@@ -220,3 +220,7 @@ _UI_MavenProjectConfiguration_groupId_feature = Group Id
 _UI_MavenProjectConfiguration_artifactId_feature = Artifact Id
 _UI_MavenProjectConfiguration_version_feature = Version
 _UI_MavenServerConfiguration_jettyPort_feature = Jetty Port
+_UI_ResourceProperty_rdfCollectionRepresentation_feature = Rdf Collection Representation
+_UI_ResourcePropertyRdfCollectionRepresentation_n_a_literal = n_a
+_UI_ResourcePropertyRdfCollectionRepresentation_rdfList_literal = rdfList
+_UI_ResourcePropertyRdfCollectionRepresentation_rdfBag_literal = rdfBag
```

Comment:
how about 'rdf:Bag' or "RDF 'Bag' Container"

Reply:
ok. I'll fix the display.